### ISSUE TITLE
What Ip address to hit from web client

### ISSUE THREAD

**Issue by tim-br** [ledgerlabs/token#1]
On line 96:

`web3.setProvider(new web3.providers.HttpProvider('http://159.203.28.215:8545'));`,

the ip address should be set to whatever ip address the local node is being served from?

**Comment by featheredape** [comment]
Yes, as long you have --rpc etc enabled.

That address might be Nick's server, and might not be up

**Comment by SilentCicero** [comment]
Yeah, that DO node is a crashy clam. Switching to better backend (i.e. Azure/AWS)...

**Comment by tim-br** [comment]
It WAS working with Nick's node, and I was wondering what I was doing to enable that, it seemed like magic.

Is there a way to start all that rpc stuff and unlock account from `geth --testnet console`, I can't open the console at the same time as when I have that rpc stuff open, but it's quite handy to have the console.
